nereid/nereid/src/network/utils.py

```diff
@@ -93,6 +93,36 @@ def thin_graph_dict(graph_dict: Dict[str, Any]) -> Dict[str, Any]:
 
 
 def nxGraph_to_dict(g: nx.Graph) -> Dict[str, Any]:
+    """Convert a networkx garph object into a dictionary
```

Comment:
maybe someday. i'm generally not a fan of doctests... nothing against them, just hate fiddling with code that lives its whole life as a comment.

plus this function is tested to what i consider an 'overkill' degree since things like 'multi graph' and 'undirected graph' support are very much considered 'invalid watersheds' for our purposes.

Reply:
That's fair. What I do like about doctests is that they ensure your docs are correct. "wrongs docs are worse than no docs" and whatnot